test/test_mps.py

```diff
@@ -9862,7 +9862,7 @@ class TestConsistency(TestCase):
 
         # failures due to issue #102048039: powerWithPrimaryTensor() with integer input may return wrong results
         'pow': [torch.int16, torch.int32, torch.int64, torch.uint8],
-        '__rpow__': [torch.int16, torch.int32, torch.uint8, torch.int64],
+        '__rpow__': [torch.uint8],
```

Comment:
I think we should move all the types to Blocklist, we might be just getting lucky if the uint8 works.

Reply:
As uint8 is unsigned, there is no issue of 0 power of negative numbers.
 
On macos12:
	square: float16, float32 work
	pow: float16 work
	__rpow__: none work

One macos13 without fast math off:
	square: all good
	pow: float32 fail
	rpow: uint8 work

One maco13 with fast math off:
	square: all good
	pow: all good
	rpow: uint8, float32 work

In fact, after the fast math off patch goes into new system, we should move this to allow list, and add them to macos12 block list.